v2/internal/api_test.go

```diff
@@ -391,13 +365,7 @@ func launchHelperProcess(t *testing.T) (cleanup func()) {
 		t.Fatal("Helper process never reported")
 	}
 
-	restoreAPIHost := restoreEnvVar("API_HOST")
```

Comment:
I'm spoiled by Critique. Github's diff view is pretty hard to read for this type of refactors. I couldn't figure out why these lines can be deleted, or if they are replaced elsewhere.

Reply:
They're replaced by RegisterTestRequest which is called right after this launchHelperProcess function on line #309. RegisterTestRequest does the same setup/teardown, but using context.Context.Values instead of with environment variables.